sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -119,6 +119,19 @@ static const struct dmi_system_id sof_rt5682_quirk_table[] = {
 		.driver_data = (void *)(SOF_RT5682_MCLK_EN |
 					SOF_RT5682_SSP_CODEC(0)),
 	},
+	{
```

Comment:
@sathya-nujella there's a typo in the commit message "overiride"

Reply:
oh, sorry for that typo. fixed it now.